inca/rssscrapers/UK_dailystar_scraper.py

```diff
@@ -55,32 +55,42 @@ def parsehtml(self, htmlsource):
             logger.warning("Cannot parse HTML tree", type(doc), len(doc))
             # logger.warning(doc)
             return ("", "", "", "")
+        
         try:
-            title = tree.xpath("//*[@itemprop='headline']//text()")[0]
+            title = " ".join(tree.xpath("//*[@itemprop='headline name']//text()"))
         except:
             title = ""
             logger.warning("Could not parse article title")
+        
         try:
-            teaser = tree.xpath("//*[@id='singleArticle']/header/p/text()")[0]
+            teaser = " ".join(tree.xpath("//*[@itemprop='description']//text()"))
         except:
             teaser = ""
             logger.debug("Could not parse article teaser")
+        
         try:
             byline = tree.xpath("//*[@class='author']//text()")[0]
         except:
             byline = ""
             logger.debug("Could not parse article source")
+        
         try:
-            text = " ".join(tree.xpath("//*[@data-type='article-body']//p/text()"))
+            text = " ".join(tree.xpath("//*[@itemprop='articleBody']//p/text()"))
         except:
             text = ""
             logger.warning("Could not parse article text")
+        
+        try:
+            category = " ".join(tree.xpath('//*[@class="breadcrumbs breadcrumbs-news"]//li[2]/a/span/text()'))                                         
+        except:
+            category = ""
```

Comment:
added category but realized that this is only scraper for latest news from dailystar.. is that intended? if so, category = redundant, I guess